sdk/jobs/pipelines/2e_image_classification_keras_minist_convnet/image_classification_keras_minist_convnet.ipynb

```diff
@@ -138,19 +150,54 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "prepare_dataset = load_component(yaml_file='./keras-mnist-fashion/prepare.yaml')\n",
-    "train_model = load_component(yaml_file='./keras-mnist-fashion/train.yaml')"
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "# if you modify the componetn source code, use following code to reload it\n",
+    "# otherwise notebook has cache and may not load the latest update\n",
+    "import importlib, prep.prep_dsl_component, train.train_dsl_component\n",
+    "importlib.reload(prep.prep_dsl_component)\n",
```

Comment:
Maybe we add another section which comments these code as they are not a must. 

Hint user :
Python import will have cache if their code not change. They might need a force re-import when only conda changes.